### ISSUE TITLE
[MobileNet.js] operator type 43 is not supported by using new mobilenet_v1_1.0_224.tgz

### ISSUE THREAD

**Issue by ibelem** [intel/webml-polyfill#106]
Based on https://github.com/tensorflow/tensorflow/commit/28d43e5ada3c1e16b81c64b08cbbc273407a0347?short_path=7eb6a41#diff-7eb6a41d42575d4116fee4b816c60341 , all versions of mobilenet were updated to .tgz format from .zip with different contents:

```
├── mobilenet_v1_1.0_224_float_2017_11_08.zip
│   ├── labels.txt
│   └── mobilenet_v1_1.0_224.tflite

├── mobilenet_v1_1.0_224.tgz
│   ├── mobilenet_v1_1.0_224.ckpt.data-00000-of-00001
│   ├── mobilenet_v1_1.0_224.ckpt.index
│   ├── mobilenet_v1_1.0_224.ckpt.meta
│   ├── mobilenet_v1_1.0_224_eval.pbtxt
│   ├── mobilenet_v1_1.0_224_frozen.pb
│   ├── mobilenet_v1_1.0_224_info.txt
│   └── mobilenet_v1_1.0_224.tflite
```

It seems like the .ckpt.data-00000-of-00001, .ckpt.index and .ckpt.meta are saved by tf.train.Saver(), and there is no labels.txt file in the .tgz anymore.

Repro Steps:
1. Download http://download.tensorflow.org/models/mobilenet_v1_2018_02_22/mobilenet_v1_1.0_224.tgz
2. Upzip mobilenet_v1_1.0_224.tgz
3. Copy unzipped mobilenet_v1_1.0_224.tflite to ~/webml-polyfill/examples/mobilenet/model
4. keep 1.0 labels.txt from mobilenet_v1_1.0_224_float_2017_11_08.zip under ~/webml-polyfill/examples/mobilenet/model
5. npm start
6. Visit http://127.0.0.1:8081/examples/mobilenet/

Actual Result:
```
Uncaught (in promise) Error: operator type 43 is not supported.
    at MobileNet._addOpsAndParams (MobileNet.js:193)
    at MobileNet.createCompiledModel (MobileNet.js:36)
    at <anonymous>
```

There is no 43 in our operator types.

**Comment by GreyZzzzzzXh** [comment]
There are more operator types supported in TFLite([https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/builtin_ops](url)).
Type43 squeeze is used now, while the old one used reshape(type 22).

**Comment by Wenzhao-Xiang** [comment]
Bug fixed. We can run **mobilenet_v1_1.0_224** in mobilenet example now.

Notes :

- ops '**squeeze**' is used for removing dimensions of size 1 from the shape of a inputTensor.
(Reference: [tf.squeeze (x, axis?)](https://js.tensorflow.org/api/0.9.0/#squeeze))

- In **mobilenet_v1_1.0_224**, squeeze is used to change the inputShape from [1,1,1,1001] to [1,1001],and just has one input here.So we use ‘reshape’ to replace it equivalently by adding an extra oprand 'targetShape' and giving it to inputs of 'reshape'.

**Comment by Wenzhao-Xiang** [comment]
It seems that **mobilenet_v1_1.0_224** is almost as fast as the last model.Its accuracy has some difference with the last model,but I think It's acceptable.

Here is the accuracy comparison:

**mobilenet_v1_1.0_224**
‘bee eater’:  95.59%
‘pembroke’: 93.04%
**mobilenet_v1_1.0_224_float_2017_11_08**
‘bee eater’:  99.71%
‘pembroke’: 92.20%

**Comment by ibelem** [comment]
@huningxin The PR #154 is still in open status, we need to verify the bug when PR is merged (standard process).

**Comment by huningxin** [comment]
@BruceDai , please help verify. Thanks!

**Comment by Christywl** [comment]
Checked it with the chromium nightly build(e7980af), this issue wasn't reproduced.
